Automattic/wp-calypso
# Emails: Show suspended account on suspended Professional Email accounts

#### Changes proposed in this Pull Request

This pull request adds a small error icon when there is any email account suspended for Professional Email.
I am setting this as a draft as I would like to know if we should keep allowing the user to Add new mailboxes/manage them. If so I will grey out/remove those options.

#### Testing instructions

1. Checkout locally this branch
2. Have (or create) Professional Email subscription
3. Sandbox your public api
4. Follow these instructions: 2befc-pb
5. Check that you see those error notices in the email management section and also in the emails Home Screen.

| Emails home  | Email management |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/5689927/158181655-6659da50-2287-4cb7-9044-2599bed9a21e.png) | ![image](https://user-images.githubusercontent.com/5689927/158181545-d60a9db3-8bef-42d9-b877-0ea4e28427ed.png) |


Related to {1200182182542585-as-1201373094310942}
